packages/plugin-e2e/src/models/ExplorePage.ts

```diff
@@ -17,32 +17,50 @@ export class ExplorePage extends GrafanaPage {
   datasource: DataSourcePicker;
   timeRange: any;
   timeSeriesPanel: Panel;
-  logsPanel: Panel;
   tablePanel: Panel;
 
   constructor(ctx: PluginTestCtx) {
     super(ctx);
     this.datasource = new DataSourcePicker(ctx);
     this.timeRange = new TimeRange(ctx);
-    this.timeSeriesPanel = new Panel(this.ctx, () =>
-      this.getPanelLocators(TIME_SERIES_PANEL_SELECTOR_SUFFIX, TIME_SERIES_PANEL_TEXT)
-    );
-    this.tablePanel = new Panel(this.ctx, () => this.getPanelLocators(TABLE_PANEL_SELECTOR_SUFFIX, TABLE_PANEL_TEXT));
-    this.logsPanel = new Panel(this.ctx, () => this.getPanelLocators(LOGS_PANEL_SELECTOR_SUFFIX, LOGS_PANEL_TEXT));
+    this.timeSeriesPanel = new Panel(this.ctx, () => this.getTimeSeriesPanelLocator());
+    this.tablePanel = new Panel(this.ctx, () => this.getTablePanelLocator());
   }
 
-  private getPanelLocators(suffix: string, text: string) {
-    const page = this.ctx.page;
-    let locator = this.getByTestIdOrAriaLabel(this.ctx.selectors.components.Panels.Panel.title(suffix), {
-      startsWith: true,
-    });
+  private getTablePanelLocator() {
```

Comment:
locating table panel and time series panel in explore is a mess. for some reason, selectors seem to change in every version of Grafana. need to look into this at some point and investigate how this can be improved.